src/components/CalendarCell/CalendarCell.tsx

```diff
@@ -29,17 +29,15 @@ interface DefaultProps {
   stretchable: boolean;
   current: boolean;
   alignment: Alignment;
-  timeType: Times;
 }
 
 /** CalendarCell */
 export class CalendarCell extends React.Component<CalendarCellProps> {
   static displayName = 'CalendarCell';
   static defaultProps: DefaultProps = {
     stretchable: false,
-    current: true,
+    current: false,
```

Comment:
This is a breaking change, and we usually wouldn't do something like that.
But since you're the only one using this component that's ok 😁

Reply:
sorry >_<"